airbyte-integrations/connectors/source-google-drive/source_google_drive/source.py

```diff
@@ -14,17 +14,13 @@
 from source_google_drive.stream_reader import SourceGoogleDriveStreamReader
 
 
-class SourceGoogleDrive(FileBasedSource):
-    def __init__(self, catalog: Optional[ConfiguredAirbyteCatalog], config: Optional[Mapping[str, Any]], state: Optional[TState]):
-        super().__init__(
-            stream_reader=SourceGoogleDriveStreamReader(),
-            spec_class=SourceGoogleDriveSpec,
-            catalog=catalog,
-            config=config,
-            state=state,
-            cursor_cls=DefaultFileBasedCursor,
-            stream_permissions_reader=SourceGoogleDriveStreamPermissionsReader(),
-        )
+class SourceGoogleDrive():
```

Comment:
Is the missing base/parent class intended? (FileBasedSource)

Reply:
Weird. No. Let me fix that real quick.

```suggestion
class SourceGoogleDrive(FileBasedSource):
```